TF-201: Tray icon menu starts and stops touch free

NatGreenwood-UL opened the pull request:
- [x] Test plan updated (by PR creator) [TF-557]
- [x] Code Review 
- [x] Developer Testing: Key item tests 
- [x] Developer Testing: Execution of the updated test plan (changed items for this work only) 
- [x] Product Owner Review (Optional) 
- [x] Update Release Notes on the JIRA ticket for harvest at release time

NatGreenwood-UL committed:
First pass of changing the name of the item and stopping if it's already
running

NatGreenwood-UL committed:
Make CheckForTouchFree suitable for the timer event

NatGreenwood-UL committed:
Complete removal of optional intialisation as TF is now always installed

MattGrayUL commented on TF_Service_Utilities/ServiceUITray/ServiceUITray/Program.cs:
```diff
@@ -47,62 +49,39 @@ public class ServiceUITray : ApplicationContext
+                ContextMenu = new ContextMenu(new MenuItem[] {
+                    touchFreeMenuItem,
+                    new MenuItem("-"),
+                    new MenuItem("Service Settings", Settings),
```
Unless this has changed recently, I believe this should just say 'Settings' rather than 'Service Settings'?

NatGreenwood-UL committed:
Merge branch 'epic/TF-146-Unified_TF_Service_and_App_Settings' into feature/TF-201_Tray_Icon_Menu_starts_and_stops_TouchFree

NatGreenwood-UL committed:
Merge branch 'epic/TF-146-Unified_TF_Service_and_App_Settings' into feature/TF-201_Tray_Icon_Menu_starts_and_stops_TouchFree

NatGreenwood-UL committed:
Stop detection of TF getting stuck in "it was once found so don't
bother"

Also fix the application name reference and switch from close() to
kill() for the TF process, same as on Tray icon exit.
